### DISCUSSION TITLE
Pocketbase restarts and deletes all collections

### DISCUSSION THREAD

**Discussion by javalen** [pocketbase/pocketbase#3614] [Q&A]
I've deployed pocketbase on Fly.io successfully. I then create a new user and a new collection. Approximately 10 minutes late I revisit the pocketbase dashboard and get the message "Create your first admin account in order to continue". I create a new admin user, login and all of the data I previously created is gone. Any help would be greatly appreciated.

**Comment by ganigeorgiev** [comment]
It seems that you didn't mount a persistent volume for the `pb_data` directory.

It could be a little outdated, but for an example Fly guide you can check https://github.com/pocketbase/pocketbase/discussions/537.

If the above doesn't help, feel free to share your fly.toml and Dockerfile and I'll give it a try.
